component/ci-light/BUCK

```diff
@@ -0,0 +1,8 @@
+load(
+    "@prelude-si//:macros.bzl",
+    "docker_image",
+)
+
+docker_image(
+    name = "ci-light",
+)
```

Comment:
Might be worth extracting the version of the `rust` image (that is the `1-alpine` in this case) into a `build_args` entry so we can bump this version here, much like in the NATS [BUCK file](https://github.com/systeminit/si/blob/main/component/nats/BUCK) and [Dockerfile](https://github.com/systeminit/si/blob/main/component/nats/Dockerfile) setup.